crypto/cpu-aarch64-linux.c

```diff
@@ -19,13 +19,51 @@
 
 #include <sys/auxv.h>
 
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
+#endif
+#include <inttypes.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
 #include <openssl/arm_arch.h>
 
 #include "internal.h"
 
 
 extern uint32_t OPENSSL_armcap_P;
 
+// handle_cpu_env applies the value from |in| to the CPUID values in |out[0]|
```

Comment:
Could this comment include example usage?
To set up 2 capabilities, how is it done? I'm guessing by putting directly the value that represents both.
How can `~` and `|` be used?

Reply:
done